mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -196,4 +196,18 @@ object Statistics {
     : KolmogorovSmirnovTestResult = {
     KolmogorovSmirnovTest.testOneSample(data, distName, params: _*)
   }
+
+  /**
+   * Perform a two-sample, two-sided Kolmogorov-Smirnov test for probability distribution equality
+   * The null hypothesis corresponds to both samples coming from the same distribution
```

Comment:
"corresponds" is a little weird.  Maybe just "The null hypothesis is that both samples come from the same distribution".